lib/cocoapods-core/specification.rb

```diff
@@ -255,7 +255,7 @@ def recursive_subspecs
     #
     # @return   [Specification] the subspec with the given name or self.
     #
-    def subspec_by_name(relative_name, raise_if_missing = true)
+    def subspec_by_name(relative_name, raise_if_missing = true, include_test_specications = false)
```

Comment:
Cool, @DanToml and I had a chat about this yesterday as we tried to figure out if it would break external projects (e.g. Jazzy) and it doesn't look like it - sounds 👍